packages/transaction-controller/src/utils/simulation.test.ts

```diff
@@ -294,58 +311,98 @@ describe('Simulation Utils', () => {
 
     describe('returns token balance changes', () => {
       it.each([
-        [
-          'ERC-20 token',
-          PARSED_ERC20_TRANSFER_EVENT_MOCK,
-          SimulationTokenStandard.erc20,
-          undefined,
-        ],
-        [
-          'ERC-721 token',
-          PARSED_ERC721_TRANSFER_EVENT_MOCK,
-          SimulationTokenStandard.erc721,
-          TOKEN_ID_MOCK,
-        ],
-        [
-          'ERC-1155 token via single event',
-          PARSED_ERC1155_TRANSFER_SINGLE_EVENT_MOCK,
-          SimulationTokenStandard.erc1155,
-          TOKEN_ID_MOCK,
-        ],
-        [
-          'ERC-1155 token via batch event',
-          PARSED_ERC1155_TRANSFER_BATCH_EVENT_MOCK,
-          SimulationTokenStandard.erc1155,
-          TOKEN_ID_MOCK,
-        ],
-      ])('on %s', async (_title, parsedEvent, standard, id) => {
-        mockParseLog({ [standard]: parsedEvent });
-
-        simulateTransactionsMock
-          .mockResolvedValueOnce(
-            createEventResponseMock([createLogMock(CONTRACT_ADDRESS_MOCK)]),
-          )
-          .mockResolvedValueOnce(
-            createBalanceOfResponse([BALANCE_1_MOCK], [BALANCE_2_MOCK]),
-          );
+        {
+          title: 'ERC-20 token',
+          parsedEvent: PARSED_ERC20_TRANSFER_EVENT_MOCK,
+          tokenType: SupportedToken.ERC20,
+          tokenStandard: SimulationTokenStandard.erc20,
+          tokenId: undefined,
+          previousBalances: [BALANCE_1_MOCK],
+          newBalances: [BALANCE_2_MOCK],
+        },
+        {
+          title: 'ERC-721 token',
+          parsedEvent: PARSED_ERC721_TRANSFER_EVENT_MOCK,
+          tokenType: SupportedToken.ERC721,
+          tokenStandard: SimulationTokenStandard.erc721,
+          tokenId: TOKEN_ID_MOCK,
+          previousBalances: [OTHER_ADDRESS_MOCK],
+          newBalances: [USER_ADDRESS_MOCK],
+        },
```

Comment:
Related to previous comment: Can you help me understand how an address relates to a balance?

Reply:
See other comment.